vspace/src/main/java/edu/asu/diging/vspace/web/staff/EditImageController.java

```diff
@@ -0,0 +1,62 @@
+package edu.asu.diging.vspace.web.staff;
+
+import java.io.FileNotFoundException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+import edu.asu.diging.vspace.core.data.ImageRepository;
+import edu.asu.diging.vspace.core.exception.FileStorageException;
+import edu.asu.diging.vspace.core.factory.IImageFactory;
+import edu.asu.diging.vspace.core.model.impl.VSImage;
+import edu.asu.diging.vspace.web.staff.forms.ImageForm;
+
+@Controller
+public class EditImageController {
+
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+    
+    @Autowired
+    private ImageRepository imageRepo;
```

Comment:
your controller should not depend on the repository, just on the manager. Your manager is the connection to the repository to keep it loosely coupled.